### PR TITLE
Feature/seed 1006 add request failure and retries

### PR DESCRIPTION
**Pull request by jpaidoussi**

### CODE REVIEW

File: message_sender/tasks.py [hunk 1]
```diff
@@ -180,94 +192,138 @@ def run(self, message_id, **kwargs):
...
+            else:
+                retry_delay = self.default_retry_delay
+            l.info("Attempts: %s" % message.attempts)
+            # send or resend
+            try:
+                if "voice_speech_url" in message.metadata:
+                    # OBD number of tries metric
                     fire_metric.apply_async(kwargs={
-                        "metric_name": 'vumimessage.tries.sum',
+                        "metric_name": 'vumimessage.obd.tries.sum',
```
Comment (smn) [inline message_sender/tasks.py:210]: Does anything reading this metric need to be changed now?

Comment (jpaidoussi) [reply message_sender/tasks.py:210]: I think this is a quirk of how the diff is being shown. I didn't change the metric names.

Comment (smn) [reply message_sender/tasks.py:210]: ah ok!

**Comment by smn** [comment]
:+1: from me

**Pull request merged**